Description: snmpd.conf.5 cannot extend EXTEND
 Remove line in man page that you can extend EXTEND with
 SNMP SETs
Author: Craig Small <csmall@debian.org>
Origin: Debian
Reviewed-by: Craig Small <csmall@debian.org>
Last-Update: 2020-07-31
--- a/man/snmpd.conf.5.def
+++ b/man/snmpd.conf.5.def
@@ -1308,9 +1308,6 @@
 \fIrun-command(3)\fR.  Unlike the equivalent \fIexecfix\fR,
 this directive does not need to be paired with a corresponding
 \fIextend\fR entry, and can appear on its own.
-.PP
-Both \fIextend\fR and \fIextendfix\fR directives can be configured
-dynamically, using SNMP SET requests to the NET\-SNMP\-EXTEND\-MIB.
 .SS "MIB-Specific Extension Commands"
 The first group of extension directives invoke arbitrary commands,
 and rely on the MIB structure (and management applications) having
